com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -1696,13 +1696,13 @@ class TemporalAntiAliasingData
             }
             passData.destination = builder.WriteTexture(dest);
 
-            bool needToUseCurrFrameSizeForHistory = camera.resetPostProcessingHistory || TAAU != camera.previousFrameWasTAAUpsampled;
+            bool needToUseCurrFrameSizeForHistory = camera.resetPostProcessingHistory || upscalingEnabled != camera.previousFrameWasTAAUpsampled;
```

Comment:
This condition now reads a bit weird 

upscalingEnabled != camera.previousFrameWasTAAUpsampled  I assume will always be true with DLSS on, no?